systemtests/tests/virtualfull-deletedfiles/testrunner-05-consolidate

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+
+TestName="$(basename "$(pwd)")"
+export TestName
+
+myname=$(basename "$0")
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+
+start_test
+
+find "${tmp}/data/" -type f -delete
+echo "$0" >"${tmp}/data/$myname"
+
+cat <<END_OF_DATA >"$tmp/bconcmds"
+@$out /dev/null
+messages
+@$out $tmp/$myname.log
+run job=backup-bareos-fd level=VirtualFull jobid=3,4 yes
+wait
+messages
+quit
+END_OF_DATA
+
+run_bconsole
+
+# we expect 4 files: the added file, the changed directory and from each of the
+# two jobs a deleted file
+if ! grep -q 'SD Files Written:.*4' "$tmp/$myname.log"; then
+  echo "Number of files mismatch!"
+  exit 2
+fi
```

Comment:
This test fails even with the quick fix.  This test should probably be moved to the real fix and replaced by another test that tests the fixes of the quick fix.